website/docs/r/virtual_hub_connection.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_virtual_hub_connection"
+description: |-
+  Manages a Connection for a Virtual Hub.
+---
+
+# azurerm_virtual_hub_connection
+
+Manages a Connection for a Virtual Hub.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+resource "azurerm_virtual_network" "example" {
```

Comment:
could we put new lines between resources
```suggestion

resource "azurerm_virtual_network" "example" {
```

Reply:
Updated